diff --git a/node_modules/react-native/Libraries/Components/ScrollView/.ScrollView.js.swp b/node_modules/react-native/Libraries/Components/ScrollView/.ScrollView.js.swp
new file mode 100644
index 0000000..2c16628
Binary files /dev/null and b/node_modules/react-native/Libraries/Components/ScrollView/.ScrollView.js.swp differ
diff --git a/node_modules/react-native/Libraries/Image/.ImageBackground.js.swp b/node_modules/react-native/Libraries/Image/.ImageBackground.js.swp
new file mode 100644
index 0000000..1d445eb
Binary files /dev/null and b/node_modules/react-native/Libraries/Image/.ImageBackground.js.swp differ
diff --git a/node_modules/react-native/Libraries/Pressability/.Pressability.js.swp b/node_modules/react-native/Libraries/Pressability/.Pressability.js.swp
new file mode 100644
index 0000000..70a1a18
Binary files /dev/null and b/node_modules/react-native/Libraries/Pressability/.Pressability.js.swp differ
diff --git a/node_modules/react-native/Libraries/Renderer/implementations/.ReactNativeRenderer-dev.js.swp b/node_modules/react-native/Libraries/Renderer/implementations/.ReactNativeRenderer-dev.js.swp
new file mode 100644
index 0000000..d42253c
Binary files /dev/null and b/node_modules/react-native/Libraries/Renderer/implementations/.ReactNativeRenderer-dev.js.swp differ
diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index 7149c64..6bb0079 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -410,13 +410,6 @@ module.exports = {
     return require('deprecated-react-native-prop-types').PointPropType;
   },
   get ViewPropTypes(): $FlowFixMe {
-    console.error(
-      'ViewPropTypes will be removed from React Native, along with all ' +
-        'other PropTypes. We recommend that you migrate away from PropTypes ' +
-        'and switch to a type system like TypeScript. If you need to ' +
-        'continue using ViewPropTypes, migrate to the ' +
-        "'deprecated-react-native-prop-types' package.",
-    );
     return require('deprecated-react-native-prop-types').ViewPropTypes;
   },
 };
